backend/app/Models/Following.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Following extends Model
+{
+    use HasFactory;
+
+  public function student()
+  {
+    return $this->belongsTo('App\Models\Student');
```

Comment:
Be consistent, instead of:
```suggestion
    return $this->belongsTo(Student::class);
```

Reply:
Noted Sir.